_providers/disroot.md

```diff
@@ -3,6 +3,7 @@ name: Disroot
 status: OK
 domains: 
   - disroot.org
+strict_tls: 1
```

Comment:
`true` everywhere but `1` here?

Update script handles it correctly, but maybe better to use `true` everywhere.

Reply:
yip